kubernetes.go

```diff
@@ -211,6 +211,11 @@ func (c *KubernetesClient) getServices() ([]ServiceInformation, error) {
 				continue
 			}
 
+			if InEphemeralPortsRange(s.Spec.Ports) {
+				log.Printf("Service %s in %s has a Service Port within the ephemral ports range (%s), skipping it. Please check your configuration!.", s.Name, s.Namespace, ephemeralPortsRange.String())
```

Comment:
No need to call `String()`, now your object implements the [`Stringer`](https://golang.org/pkg/fmt/#Stringer) interface :)